openapi/api.yml

```diff
@@ -182,3 +182,21 @@ paths:
             application/json:
               schema:
                 $ref: 'components.yml#/components/schemas/GeneralError'
+
+  /reporting-period-range:
+    get:
+      summary: Returns the start and end date reporting period.
+      operationId: core.controllers.get_filters.get_reporting_period_range
+      responses:
+        '200':
+          description: Returns the earliest reporting period start date and the latest reporting period end date
```

Comment:
is it worth mentioning datetime format/spec?

Reply:
good shout